clog/loggers.py

```diff
@@ -239,6 +242,16 @@ def close(self):
         self.connected = False
 
 
+class MonkLogger(object):
+    """Wrapper around MonkProducer"""
+
+    def __init__(self, client_id, host, port):
+        self.producer = MonkProducer(client_id, host, port)
```

Comment:
If host and port are not provided, the monk client library will use the default ones. I think it would be better to have host and port optional, with default to None, both here and in the __init__ of MonkHandler.